templates/components/account/address-list.html

```diff
@@ -9,7 +9,7 @@
                 <div class="panel-body">
                     <h5 class="address-title">{{first_name}} {{last_name}}</h5>
                     <ul class="address-details address-details--postal">
-                        <li>{{{company}}}</li>
+                        <li>{{company}}</li>
```

Comment:
@bc-donfran company name is not escaped for a reason, please check #2399.
This is address information shopper fills in for his/her account. What type of security issue do you expect?

Reply:
@BC-krasnoshapka 

Something must of changed on customers team end as the data isn't encoded anymore. I've added a screenshot to prove this
![image](https://github.com/user-attachments/assets/5bc738e1-eb67-40d1-9453-08ba742c24d1)

Attacks so far are XSS or HTML injection. But can affect merchants that use impersonation.

Reply:
I see 2 cases:
1. Customer account on storefront (Cornerstone) - address info is entered by shopper and is viewed by shopper. So XSS or HTML injection is possible without escaping but what is the impact of it?
2. Any merchant facing UI. Should be always escaped, but not in Cornerstone.